presto-native-execution/presto_cpp/main/common/Configs.cpp

```diff
@@ -660,4 +665,14 @@ BaseVeloxQueryConfig* BaseVeloxQueryConfig::instance() {
   return instance.get();
 }
 
+void BaseVeloxQueryConfig::initialize(const std::string& filePath) {
+  ConfigBase::initialize(filePath);
+  update(*SystemConfig::instance());
```

Comment:
Do we need to update here if have called update(*SystemConfig::instance()) in ctor? Thanks!

Also can we test this in BaseVeloxQueryConfigTest? Thanks!

Reply:
Initial ctor will load from default values. SystemConfig may be initialized from a file after BaseVeloxQueryConfig is created - so we load again.